word_finder.py

```diff
@@ -0,0 +1,41 @@
+from itertools import permutations
+
+with open('collins_scrabble_words.txt', 'r') as file:
```

Comment:
In general, its a good practice to name things "semantically" -- meaning that they should be named not just for what they are intrinsically, (a file), but rather what their _purpose_ is. Something like `words_file` would be a better name for this variable.

Reply:
My thought on the naming was actually that it made clear it is Scrabble specific words.